mlir/include/Catalyst/Driver/CompilerDriver.h

```diff
@@ -55,7 +55,7 @@ typedef enum {
 struct Pipeline {
     typedef std::string Name;
     typedef llvm::SmallVector<std::string> PassList;
```

Comment:
```suggestion
    using Name =  std::string;
    using PassList = llvm::SmallVector<std::string> ;
```

Just a quick fly-by -- for CPP it is better to favour `using` rather than C-like typedef, as it allows subsequent use if/when templating arguments. For consistency, if not writing C files, `using` can be preferential always (see https://github.com/PennyLaneAI/dev_guidelines/tree/main/cpp for some additional info)